galcheat/data/README.md

```diff
@@ -0,0 +1,125 @@
+`galcheat` data files
+=====================
+
+The following document describes the parameters expected for the photometric surveys and their filters: how they should be computed, their units, etc.
+
+Survey parameters
+-----------------
+### Units and types
+
+| parameter name    | type  | units          |
+| ----------------- | ----- | -------------- |
+| name              | str   | –              |
+| pixel_scale       | float | arcsec / pixel |
+| gain              | float | e- / ADU       |
+| mirror_diameter   | float | m              |
+| obscuration       | float | dimensionless  |
+| zeropoint_airmass | float | dimensionless  |
+
+### Description
+
+#### `name`
+
+The classical name or abbreviation for the survey. Most often this is how the survey is referred to.
+In case of an ambiguity, for instance when a survey has several instruments, the name of the instrument should be added as a suffix (e.g. `Euclid_VIS`).
+
+#### `pixel_scale`
+
+Size of a square pixel on the sky.
+
+#### `gain`
+
+Conversion factor between the photo-electrons received by the camera and the digital counts after the amplification of the electronics.
+
+#### `mirror_diameter`
+
+Primary mirror diameter, in meters.
+
+#### `obscuration`
+
+Proportion of the total area of the telescope that is obscured by the position of secondary mirrors, lenses, camera, etc.
+
+This parameter is used to compute the effective area of the telescope.
+
+#### `zeropoint_airmass`
+
+Airmass value at which the zeropoint is computed.
+
+Filter parameters
+-----------------
+### Units and types
+
+| parameter name       | type      | units          |
+| -------------------- | --------- | -------------- |
+| name                 | str       | –              |
+| sky_brightness       | float     | mag / arcsec^2 |
+| exposure_time        | int/float | s              |
+| psf_fwhm             | float     | arcsec         |
+| zeropoint            | float     | e- / s         |
```

Comment:
Should the zeropoint be in `mag`? It's a magnitude not a flux right?

Reply:
Good catch, probably copy-pasted from the gain

Reply:
Nah it's because it's a magnitude for 1e-/s. And I got confused.